AngouriMath/Convenience/AngouriMathExtensions.cs

```diff
@@ -41,15 +41,15 @@ public static FiniteSet ToSet(this IEnumerable<Entity> elements)
         /// </summary>
         /// <returns>A set of unique elements</returns>
         public static Set Unite(this IEnumerable<Set> sets)
-            => sets.Aggregate((a, b) => MathS.Union(a, b));
+            => sets.Any() ? sets.Aggregate((a, b) => MathS.Union(a, b)) : Empty;
```

Comment:
Empty can be the seed: https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.aggregate?view=netcore-3.1#System_Linq_Enumerable_Aggregate__2_System_Collections_Generic_IEnumerable___0____1_System_Func___1___0___1__